api/src/main/java/ca/bc/gov/educ/studentdatacollection/api/orchestrator/SdcStudentProcessingOrchestrator.java

```diff
@@ -127,6 +127,7 @@ private void calculateAdditionalStudentAttributes(final Event event, final SdcSa
     this.sdcSchoolCollectionStudentService.updateProgramEligibilityColumns(programEligibilityErrors, studentUUID);
 
     // Calculate Fte
+    this.sdcSchoolCollectionStudentService.clearSdcSchoolStudentProgramEligibilityColumns(studentUUID);
```

Comment:
Should this go above the calculate fte comment?

Reply:
...yep. https://github.com/bcgov/EDUC-STUDENT-DATA-COLLECTION-API/pull/199/commits/b6ce1a3b316cbb8b6c1fbfd4a9e88ffc8906682f

Thank you very much.